lms/static/scripts/frontend_apps/errors.js

```diff
@@ -34,15 +33,15 @@ export class APIError extends Error {
      *
      * This can be used to show custom error dialogs for specific issues.
      */
-    this.errorCode = data.error_code || null;
+    this.errorCode = data.error_code;
```

Comment:
I'm not sure why this and `errorMessage` below were defaulting to `null`. It didn't feel right so I've let them be `undefined` if they're not present.